RetrospectiveExtension.Frontend/utilities/external/telemetryClient.ts

```
@@ -0,0 +1,17 @@
+import { ReactPlugin } from '@microsoft/applicationinsights-react-js';
+import { ApplicationInsights, SeverityLevel } from "@microsoft/applicationinsights-web"
+import environment from '../../config/environment'
+
+
+const reactPlugin = new ReactPlugin();
+const appInsights = new ApplicationInsights({
+    config: {
+        instrumentationKey: environment.AppInsightsInstrumentKey,
```

Do you know what happens if the `instrumentationKey` is empty?

---

no crashes happen and the application silently continues. If app insights is required to start the App, we can add a check to make sure that it is not empty and perhaps issue an error or warning if it is.

---

That's good enough for now, thanks!